proxmox/config_qemu.go

```diff
@@ -199,6 +201,7 @@ func (config ConfigQemu) UpdateConfig(vmr *VmRef, client *Client) (err error) {
 	configParams := map[string]interface{}{
 		"name":        config.Name,
 		"description": config.Description,
+		"bios":        config.Bios,
```

Comment:
I imagine the needs to not be included in the params if it is empty

Reply:
Good point. I've now made this conditional on `config.Bios` being non empty.